packages/exhibitor-migration-script/extra/exhibitor-migration-script/bin/dcos-exhibitor-migrate-perform

```diff
@@ -0,0 +1,86 @@
+#!/usr/bin/env python3
+import copy
+
+import requests
+import sys
+from requests import RequestException
+
+ZOOKEEPER_DATA_DIR_START = '/var/lib/zookeeper/snapshot'
+ZOOKEEPER_LOG_DIR_START = '/var/lib/zookeeper/transactions'
+LOG_INDEX_DIR_START = '/var/lib/zookeeper/transactions'
+
+ZOOKEEPER_DATA_DIR_GOAL = '/var/lib/dcos/exhibitor/zookeeper/snapshot'
+ZOOKEEPER_LOG_DIR_GOAL = '/var/lib/dcos/exhibitor/zookeeper/transactions'
+LOG_INDEX_DIR_GOAL = '/var/lib/dcos/exhibitor/zookeeper/transactions'
+
+
+class ValidationError(Exception):
+    def __init__(self, msg, exit_status=8):
+        self.msg = msg
+        self.exit_status = exit_status
+
+
+def get_config_json(response):
+    if response.status_code == 200 and response.headers['Content-Type'] == 'application/json':
+        return response.json()['config']
+    else:
+        raise ValidationError("Error reading configuration from Exhibitor", 2)
```

Comment:
could we include more info here about what was bad in response? For when our engineers are debugging this seems like it'll be sparse on info (would be useful to have `response.status_code` and `response.content`). Currently periodically in our integration tests for the exhibitor API we get empty results and if that happened here we'd have very little information to help debug...